File: assignment 10 - C# practice problems/12.cs
//Write a c# program to count the number of vowels in a given string.

using System;
namespace MyNameSpace
{
    class Program12
    {
        public static void CountVowelsInStringDriverFunction()
        {

            Console.WriteLine("Enter a string: ");
            string inputSrring = Console.ReadLine();

            var inpurArray = inputSrring.ToCharArray();
            var count = 0;

            foreach (var charecter in inpurArray)
            {
                if (charecter == 'a' || charecter == 'A' ||
                charecter == 'e' || charecter == 'E' ||
                charecter == 'i' || charecter == 'I' ||
                charecter == 'o' || charecter == 'O' ||
                charecter == 'u' || charecter == 'U')
                {
                    count += 1;
                }
            }

            Console.WriteLine("vowel count is " + count);
        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/12.js
//Write a JavaScript program to count the number of vowels in a given stringInputing.
function countVowelsInStringDriverFunction() {
    let stringInput = prompt("Enter a string");

    if (stringInput != null) {
        stringInput = stringInput.split("");

        let resultCount = 0;

        stringInput.forEach(element => {
            switch (element) {
                case 'a':
                case 'e':
                case 'i':
                case 'o':
                case 'u':
                    ++resultCount;
                    break;
            }
        });
        alert("vowel count is " + resultCount);
    }
}

countVowelsInStringDriverFunction()